drakrun/drakrun/drakpdb.py

```diff
@@ -324,7 +324,7 @@ def fetch_pdb(pdbname, guidage, destdir='.'):
 
             with tqdm(total=total_size, unit='iB', unit_scale=True) as pbar:
                 with open(dest, "wb") as f:
-                    for chunk in res.iter_content(chunk_size=1024 * 8):
+                    for chunk in res.iter_content(chunk_size=1024 * 1024 * 8):
```

Comment:
What's the reasoning behind this change? This will effectively lower the number of progress bar updates.

Reply:
Increment in download speed was the goal. For people with fast internet connection, the previous value seemed too less and capped the download a lot. 

But yeah, I did not account for effective progress bar updates, we can keep this value around 1MB I suppose. Will that be fine then?

Reply:
> For people with fast internet connection, the previous value seemed too less and capped the download a lot.

I don't think that chunk size is the issue, unfortunately (from my experience) MS symbol servers are very slow. 

I wrote a simple script to test this hypothesis. You can test it yourself:

```python
import sys
import requests
from tqdm import tqdm

with tqdm(unit='iB', unit_scale=True) as bar:
    with requests.get("https://speed.hetzner.de/1GB.bin", stream=True) as res:
        for chunk in res.iter_content(chunk_size=4096):
            bar.update(len(chunk))
```

as you can see, the chunk_size is quite low - 4KiB. I'm getting the same download speeds as with wget
(if this URL is slow for you, you may have to substitute it, this one is hosted in Germany)

Reply:
Please look at this once, This also tells that lower chunk size indeed decreases the download speed, https://github.com/tqdm/tqdm/issues/765

Reply:
> I don't think that chunk size is the issue, unfortunately (from my experience) MS symbol servers are very slow.

MS symbol servers are slow, that I agree, But even if some of the URL is fast enough, it will be capped due to the small chunk_size

Reply:
>This also tells that lower chunk size indeed decreases the download speed, tqdm/tqdm#765

Yes, but it only concerns using tqdm as wrapper for iterable.
Using `.update()` with 4096 bytes, I'm getting about 100MiB/s, so I still don't think that the chunk size is an issue

Reply:
Yes, I just noticed. I reverted the change